build.gradle

```diff
@@ -1643,19 +1700,6 @@ project(':server-common') {
     }
   }
 
-  sourceSets {
-    main {
-      java {
-        srcDirs = ["src/main/java"]
-      }
-    }
-    test {
-      java {
-        srcDirs = ["src/test/java"]
-      }
-    }
-  }
```

Comment:
Redundant and unnecessary.

Reply:
Should we create a ticket to get rid of this from all sub-projects? If you like, we can do it as part of this PR itself.

Reply:
I can file a ticket. Some of them are actually necessary as they add additional directories.